boards/arm/rp2040/raspberrypi-pico/configs/smp/defconfig

```diff
@@ -37,18 +37,14 @@ CONFIG_RAM_SIZE=270336
 CONFIG_RAM_START=0x20000000
 CONFIG_READLINE_CMD_HISTORY=y
 CONFIG_RR_INTERVAL=200
-CONFIG_SMP=y
```

Comment:
this is smp defconfig, why remove SMP?

Reply:
> this is smp defconfig, why remove SMP?

is a mistake I made, fixed it with another way, please help take a review again